.sync/Version.njk

```diff
@@ -30,7 +30,7 @@
 #}
 
 {# The git ref value that files dependent on this repo will use. #}
-{% set mu_devops = "v7.1.0" %}
+{% set mu_devops = "v7.2.0" %}

```

Comment:
is this going to break until 7.2.0 is released?

Reply:
No. The version will roll to 7.2.0 when this is merged due to the `type:feature-request` label being present.

Reply:
I see the plan, merge and immediately release then let it be mirrored over to the repos.

Reply:
If another PR unexpectedly rolls the same version component before this one it would throw off the version predicted here, but there's not a risk of that happening at the moment.